### PR TITLE
Fix bug with csrf token issue #68

### PR DESCRIPTION
**Pull request by mbv**
Form insertion image can have 2, 3 or 4 tabs. If it has 2 or 3 tabs, tab of uploading image is 2.  If it has 4 tabs, tab of uploading image is 3.
image
image
image

### CODE REVIEW

**Comment by nirvana-msu** [comment]
This PR should not be merged. It is not generic enough - it still does not handle multiple widgets. Instead, [the following code](https://github.com/2amigos/yii2-ckeditor-widget/issues/2#issuecomment-237703590) should be used.

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,12 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 install:
   - composer self-update
-  - composer global require "fxp/composer-asset-plugin:1.1.2"
+  - composer global require "fxp/composer-asset-plugin:~1.1.1"
```
Comment (tonydspaniard) [inline .travis.yml]: Why drop a version? 1.1.2 has fixed the speed on installs if i recall it well.

Comment (mbv) [inline .travis.yml]: I set minimum version to 1.1.1. On 1.1.2 I had an error, maybe composer broke api again. https://github.com/yiisoft/yii2/issues/10554#issuecomment-171623038

**Comment by mbv** [comment]
@nirvana-msu It works good when your locale is English. But this didn't work on other locale, when title isn't "Upload".
image
image

**Comment by nirvana-msu** [comment]
@mbv, thanks! Updated my issue comment as well.

**Comment by tonydspaniard** [comment]
Thanks all @mbv @nirvana-msu !

**Pull request merged**